loadable_apps/micomapp/micomapp.c

```diff
@@ -22,16 +22,86 @@
 
 #include <stdio.h>
 #include <unistd.h>
+#ifdef CONFIG_EXAMPLES_MICOM_TIMER_TEST
+#include <string.h>
+#endif
 #ifdef CONFIG_BINARY_MANAGER
 #include <binary_manager/binary_manager.h>
 #endif
 #include "micomapp_internal.h"
 
+#ifdef CONFIG_EXAMPLES_MICOM_TIMER_TEST
+#define TIMER_ARG_MAX 10
+#define TIMER_ARG_NUM 5
+
+char *timer_args[TIMER_ARG_NUM];
+static void alloc_timer_args(void)
+{
+	timer_args[0] = (char *)malloc(strlen("timer") + 1);
```

Comment:
I think we can remove duplicated codes. How about this?
- use `strdup` or `strndup` function.
- initiate NULL and free if NULL
```c
for (int i = 0; i < TIMER_ARG_NUM; i++) {
    timer_args[i] = NULL;
}

timer_args[0] = strdup("timer");
if (timer_args[0] == NULL) {
    goto timer_errout;
}

...

return OK;

timer_errout:
for (int i = 0; i < TIMER_ARG_NUM; i++) {
    if (timer_args[i] != NULL) {
        free(timer_args[i]);
        timer_args[i] = NULL;
    }
}
return ERROR;
```